RecoHI/HiEvtPlaneAlgos/src/HiEvtPlaneFlatProducer.cc

```diff
@@ -167,74 +205,88 @@ void HiEvtPlaneFlatProducer::produce(edm::Event& iEvent, const edm::EventSetup&
       }
     }
   }
-
+  //
+  //Get flattening parameter file.
+  //
   if (hirpWatcher.check(iSetup)) {
     edm::ESHandle<RPFlatParams> flatparmsDB_;
     iSetup.get<HeavyIonRPRcd>().get(flatparmsDB_);
     LoadEPDB db(flatparmsDB_, flat);
-    if (!db.IsSuccess())
-      return;
-  }
+  }  //rp record change
+
   //
   //Get Centrality
   //
-
-  int bin = iEvent.get(centralityBinToken) / CentBinCompression_;
-
-  if (Noffmin_ >= 0) {
-    int nOff = iEvent.get(centralityToken).Ntracks();
-    if ((nOff < Noffmin_) or (nOff >= Noffmax_)) {
-      return;
-    }
-  }
+  int bin = 0;
+  int cbin = 0;
+  edm::Handle<int> cbin_;
+  iEvent.getByToken(centralityBinToken, cbin_);
+  cbin = *cbin_;
+  bin = cbin / CentBinCompression_;
+  //double centval = cbin*100./(double) nCentBins_;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant